TestSuite/pack.py

```diff
@@ -16,6 +16,7 @@
     XSIAM_DASHBOARDS_DIR,
     XSIAM_REPORTS_DIR,
 )
+from demisto_sdk.commands.content_graph.objects.test_playbook import TestPlaybook
```

Comment:
a little suspicious that it's the only graph object imported here, let's make sure with @YuvHayun this is the right place.

Reply:
@YuvHayun WDYT?

Reply:
@YuvHayun approved